cosmoz-grouped-list.html

```diff
@@ -51,6 +52,7 @@
 				</cosmoz-grouped-list-template-selector>
 			</template>
 		</iron-list>
+		<slot name="templates" id="templates"></slot>
```

Comment:
Do we really need a specific slot to distribute the templates ?
I'm not even sure we need a slot to distribute the templates, since templates are inert elements, we don't really care if there are in the composed tree or not.